app/assets/javascripts/application.js

```diff
@@ -16,3 +16,13 @@
 //= require_tree .
 
 $(function(){ $(document).foundation(); });
+
+def reverse_string(string)
+  new_string = []
+  i = string.length-1
+  while i >= 0
```

Comment:
Expected '(' and instead saw 'i'.<br>Expected an identifier and instead saw '>='.<br>Expected ')' to match '{b}' from line 23 and instead saw '0'.